src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/common/BerConverter.cs

```diff
@@ -263,7 +262,7 @@ public static object[] Decode(string format, byte[] value)
                 throw new BerConversionException();
         }
 
-        internal static object[] TryDecode(string format, byte[] value, out bool decodeSucceeded)
+        internal static unsafe object[] TryDecode(string format, byte[] value, out bool decodeSucceeded)
```

Comment:
Why does this need `unsafe`?  None of the changes here look like they would require it.